source/install/cloud-changelog.md

```diff
@@ -4,12 +4,84 @@ This changelog summarizes updates to [Mattermost Cloud](https://mattermost.com/g
 
 Latest Mattermost Cloud releases:
 
+- [Release 2023-03-02](#release-2023-03-02)
 - [Release 2023-02-02](#release-2023-02-02)
 - [Release 2023-01-26](#release-2023-01-26)
 - [Release 2023-01-12](#release-2023-01-12)
 - [Release 2022-12-20](#release-2022-12-20)
 - [Release 2022-12-01](#release-2022-12-01)
-- [Release 2022-11-24](#release-2022-11-24)
+
+## Release 2023-03-02
+
+### Compatibility
+ - Updated Firefox minimum supported version to 102+.
+ - Updated Safari minimum supported version to 16.2+.
+ - Updated Windows minimum supported version to 10+.
+ - Updated Chromium minimum supported version to 110+.
+ - Updated Edge minimum supported version to 110+.
+
+### Highlights
+
+#### Boards
+ - Added support for person, multi-person, and date property filters in Boards.
+ - Added support for person property groups in Boards.
+ - See [the docs](https://docs.mattermost.com/boards/groups-filter-sort.html#work-with-groups-filter-and-sort) for more details.
+
+### Improvements
+
+#### User Interface (UI)
+ - Pre-packaged Calls v0.13.0.
+ - Pre-packaged Playbooks v1.36.0.
+ - All post components were removed in favor of a unified approach.
+ - App bindings are now refreshed when a App plugin enabled event gets triggered.
+ - Improvements were added to the sidebar channel and category menus.
+ - Removed right-click hijacking on code blocks in messages.
+ - The order of the Leave Channel and Archive Channel channel settings were changed to be the same as in mobile.
+ - Added the condition to remove unread styling for archived channels and to filter archived channels from local data.
+ - Changed the collapsed post fade out effect to be less buggy.
+ - Users now have the ability to see the history of edited messages and to restore an old message version with the current version.
+
+#### Administration
```

Comment:
@amyblais it looks like the mention of Cloud Pro annual was lost in the shuffle... we need to mention that Cloud Professional monthly will be no longer be offered to new customers. Cloud Pro annual will be available.

Existing Cloud Pro monthly customers will get to keep their current subscription, but can move to Cloud Pro annual if they want to

cc: @marianunez

Reply:
https://github.com/mattermost/docs/blob/aa59340622b5e1adb4620e40f212daf2b7678042/source/install/cloud-changelog.md#release-2023-02-02